extract/html-table/README.md

```
@@ -0,0 +1,61 @@
+
+- Extracts table structure, including headers and cell data
+- Handles nested tables and complex table layouts
+- Handles colspan and rowspan attributes (configurable)
+- Supports nested tables up to a configurable depth
+- Converts extracted data into a structured format
+- Provides error handling for malformed HTML or table structures
+- Debug mode for detailed logging
+
+## Parameters
+
+#### `options` - `object` - (optional)
```

Markdownlint

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)